src/index.js

```diff
@@ -30,7 +36,7 @@ const rulesConfig = {
 
 const plugin = {
     meta: {
-        name: "eslint-plugin-markdown",
+        name: "@eslint/markdown",
```

Comment:
We should update this test:

https://github.com/eslint/eslint-plugin-markdown/blob/923e440094e334d2a0c89c8b9d58bc20ade86373/tests/plugin.test.js#L71-L73